src/features/retreat/components/blacksmith/components/GoblinBlacksmithItems.tsx

```diff
@@ -67,7 +67,7 @@ export const GoblinBlacksmithItems: React.FC<Props> = ({ onClose }) => {
 
   const inventory = state.inventory;
 
-  const selectedItem = GOBLIN_BLACKSMITH_ITEMS[selectedName];
+  const selectedItem = GOBLIN_BLACKSMITH_ITEMS(state)[selectedName];
 
   const lessIngredients = () =>
```

Comment:
In the other PR the `less` naming was changed (yay!).  Update this one to match?